### PR TITLE
feat: allow title and tag to be offset separately

### PR DESCRIPTION
**Pull request by ryancharris**
This PR does the following:
1. Removes the `textLeftOffset` param
2. Adds `titleLeftOffset` and `taglineLeftOffset` instead
3. Updates the `README` to include the new params
4. Adds me to the `Who is using this?` section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,8 @@ This utility function accepts a config object. Available options are as follows:
 | imageWidth         |          | (number, default `1280`) SEO image width (defaults to Twitter ratio) |
 | imageHeight        |          | (number, default `669`) SEO image height (defaults to Twitter ratio) |
 | textAreaWidth      |          | (number, default `760`) width of title and tagline text areas        |
-| textLeftOffset     |          | (number, default `480`) distance from left edge to start text boxes  |
```
Comment (ryancharris) [inline README.md]: @jlengstorf For backwards compatability, we should probably keep this (now that I'm thinking about it). I'm thinking the new fields (i.e. `titleLeftOffset` and `taglineLeftOffset`) can default to `null`. Then, if either has a value, we can use that as opposed to the `textLeftOffset` value. Should just be a simple ternary on lines 81 and 95.

Comment (jlengstorf) [reply README.md]: yeah, I agree with that — there's another prop like that in here, so you can follow that pattern for consistency of you don't mind

Comment (ryancharris) [reply README.md]: @jlengstorf Added it back 👍

**Comment by ryancharris** [comment]
@jlengstorf Added the fix in 800afc5 🚀

**Comment by jlengstorf** [comment]
released as `@jlengstorf/get-share-image@0.6.0`

thank you!

**Pull request merged**